editor/src/editor/MenuWindow.cpp

```diff
@@ -80,19 +80,27 @@ MenuWindow::MenuWindow(UserInterface* ui, tgui::Gui* gui, tgui::Theme* theme) :
     grid_->addWidget(button, 2, 1);
     button->connect("Clicked", [this]() { this->ui_->openConfigWindow("config_dir", "sound"); });
 
+    button = tgui::Button::create();
+    button->setRenderer(theme_->getRenderer("Button"));
+    button->setSize(button_size);
+    button->setTextSize(button_text_size);
+    button->setText("Edit parameters");
+    grid_->addWidget(button, 3, 0);
+    button->connect("Clicked", [this]() { this->ui_->openParametersWindow(); });
+
```

Comment:
I know that it is my fault, but maybe those 7 buttons can be created using same code. E.g.:
```
auto create_button = [this](const std::string& name, const std::function<void()>& function, int i, int j) {
    auto button_size = CONF<sf::Vector2f>("button_size");
    unsigned int button_text_size = CONF<float>("label_text_size");
    button->setSize(button_size);
    button->setText(name);
    button->setTextSize(button_text_size);
    button->connect("Clicked", func);
    grid_->addWidget(button, i, j);
};

create_button("Load map", [this]() {
        this->ui_->resetMapList();
        this->gui_->get("load_window")->setVisible(true);
    }, 0, 0);
...
create_button("Edit parameters", [this]() { this->ui_->openParametersWindow(); }, 3, 0)
```